src/main/resources/static/api.yaml

```diff
@@ -1335,6 +1335,54 @@ paths:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
 
+  /patients/profile/logintype:
+    get:
```

Comment:
This is a public API, and I would say turn this to POST with the body containing a username, requestId or timestamp as nonce to avoid replay attacks.

Reply:
@ganesanarun thoughts?

Reply:
This is not a login type at all, as we don't allow the user to login with OTP, it just that they don't have a password set, needs some way of knowing whether password exists or not.

since it's not going to create any resource or something, I think it's okay to keep it GET and we can have rate limiting at gateway/network level, I think that's enough for this API.

Reply:
So this will still be a GET API isnt it?